<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>30</y>
      <width>450</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="primaryColorButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Primary Color</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="windowToggle">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icon/icons/Window Pixel-32.png</normaloff>
      <normalon>:/icon/icons/Window Full-32.png</normalon>
      <disabledoff>:/icon/icons/Window Pixel-32.png</disabledoff>
      <disabledon>:/icon/icons/Window Full-32.png</disabledon>
      <activeoff>:/icon/icons/Window Pixel-32.png</activeoff>
      <activeon>:/icon/icons/Window Full-32.png</activeon>
      <selectedoff>:/icon/icons/Window Pixel-32.png</selectedoff>
      <selectedon>:/icon/icons/Window Full-32.png</selectedon>:/icon/icons/Window Pixel-32.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="secondaryColorButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Secondary Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearLayerButton">
    <property name="geometry">
     <rect>
      <x>677</x>
      <y>323</y>
      <width>165</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Clear Layer</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icon/icons/Clear-32.png</normaloff>:/icon/icons/Clear-32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="animationSlider">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>450</y>
      <width>450</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="animationLabel">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>470</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>00:00:000</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="addlayerButton">
    <property name="geometry">
     <rect>
      <x>677</x>
      <y>277</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add Layer</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icon/icons/Plus Math-32.png</normaloff>:/icon/icons/Plus Math-32.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>277</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Remove Layer</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icons/Delete Sign-32.png</normaloff>:/icons/Delete Sign-32.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="renameButton">
    <property name="geometry">
     <rect>
      <x>787</x>
      <y>277</y>
      <width>55</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Rename layer</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icons/Rename-32.png</normaloff>:/icons/Rename-32.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="movelayerupButton">
    <property name="geometry">
     <rect>
      <x>677</x>
      <y>300</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Move  layer up</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icons/Move-Up-32.png</normaloff>:/icons/Move-Up-32.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="movelayerdownButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>300</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Move layer down</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icons/Move-Down-32.png</normaloff>:/icons/Move-Down-32.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="moveLeftButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icon/icons/Left Arrow-32.png</normaloff>:/icon/icons/Left Arrow-32.png</iconset>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveUpButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icon/icons/Up Arrow-32.png</normaloff>:/icon/icons/Up Arrow-32.png</iconset>
    </property>
    <property name="shortcut">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveDownButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icon/icons/Down Arrow-32.png</normaloff>:/icon/icons/Down Arrow-32.png</iconset>
    </property>
    <property name="shortcut">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveRightButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="uiresources.qrc">
      <normaloff>:/icon/icons/Right Arrow-32.png</normaloff>:/icon/icons/Right Arrow-32.png</iconset>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveFrameLeftButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>20</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveFrameRightButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveFrameUpButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveFrameDownButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fillCheck">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fillable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="frameTimeTB">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame time</string>
    </property>
   </widget>
   <widget class="QLabel" name="curFrameTimeLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Frametime:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menuSelect_PaintTool">
    <property name="title">
     <string>Select PaintTool</string>
    </property>
    <addaction name="actionPen"/>
    <addaction name="actionEraser"/>
    <addaction name="actionFill"/>
    <addaction name="actionLine"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
   </widget>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionImport_Picture"/>
    <addaction name="actionImport_video"/>
    <addaction name="actionImport_Settings"/>
   </widget>
   <widget class="QMenu" name="menuFrame">
    <property name="title">
     <string>Frame</string>
    </property>
    <addaction name="actionNew_Frame"/>
    <addaction name="actionCopy_Frame"/>
   </widget>
   <widget class="QMenu" name="menuAnimation">
    <property name="title">
     <string>Animation</string>
    </property>
    <addaction name="actionPlay"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSelect_PaintTool"/>
   <addaction name="menuImport"/>
   <addaction name="menuFrame"/>
   <addaction name="menuAnimation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionPen"/>
   <addaction name="actionEraser"/>
   <addaction name="actionFill"/>
   <addaction name="actionLine"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="separator"/>
   <addaction name="actionImport_Picture"/>
   <addaction name="actionImport_video"/>
   <addaction name="actionImport_Settings"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Frame"/>
   <addaction name="actionCopy_Frame"/>
   <addaction name="separator"/>
   <addaction name="actionRewind"/>
   <addaction name="actionPlay"/>
   <addaction name="actionForward"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionChange_Speed"/>
   <addaction name="actionColorOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Opened Folder-32.png</normaloff>:/icon/icons/Opened Folder-32.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Save-32.png</normaloff>:/icon/icons/Save-32.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/File-32.png</normaloff>:/icon/icons/File-32.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Save As-32.png</normaloff>:/icon/icons/Save As-32.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Đ</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Pencil-32.png</normaloff>:/icon/icons/Pencil-32.png</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Eraser-32.png</normaloff>:/icon/icons/Eraser-32.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Line-32.png</normaloff>:/icon/icons/Line-32.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Rectangle Stroked-32.png</normaloff>:/icon/icons/Rectangle Stroked-32.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Ellipse Stroked-32.png</normaloff>:/icon/icons/Ellipse Stroked-32.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/BG Color-32.png</normaloff>:/icon/icons/BG Color-32.png</iconset>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionImport_Picture">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icons/Import-32.png</normaloff>:/icons/Import-32.png</iconset>
   </property>
   <property name="text">
    <string>Import Picture</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="actionImport_video">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Film 2-32.png</normaloff>:/icon/icons/Film 2-32.png</iconset>
   </property>
   <property name="text">
    <string>Import video</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+@</string>
   </property>
  </action>
  <action name="actionImport_Settings">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icons/View Details-32.png</normaloff>:/icons/View Details-32.png</iconset>
   </property>
   <property name="text">
    <string>Import Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionNew_Frame">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Add File-32.png</normaloff>:/icon/icons/Add File-32.png</iconset>
   </property>
   <property name="text">
    <string>New Frame</string>
   </property>
  </action>
  <action name="actionCopy_Frame">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icons/Copy-32.png</normaloff>:/icons/Copy-32.png</iconset>
   </property>
   <property name="text">
    <string>Copy Frame</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Play-32.png</normaloff>:/icon/icons/Play-32.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Toggle Media Play/Pause, Media Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Pause-32.png</normaloff>:/icon/icons/Pause-32.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Toggle Media Play/Pause, Media Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Stop-32.png</normaloff>:/icon/icons/Stop-32.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Fast Forward-32.png</normaloff>:/icon/icons/Fast Forward-32.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="toolTip">
    <string>FastForward</string>
   </property>
   <property name="shortcut">
    <string>Media Next, Media Next</string>
   </property>
  </action>
  <action name="actionRewind">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Rewind-32.png</normaloff>:/icon/icons/Rewind-32.png</iconset>
   </property>
   <property name="text">
    <string>Rewind</string>
   </property>
   <property name="toolTip">
    <string>Rewind</string>
   </property>
   <property name="shortcut">
    <string>Media Previous, Media Previous</string>
   </property>
  </action>
  <action name="actionChange_Speed">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Time-32.png</normaloff>:/icon/icons/Time-32.png</iconset>
   </property>
   <property name="text">
    <string>Change Speed</string>
   </property>
   <property name="toolTip">
    <string>Change speed</string>
   </property>
  </action>
  <action name="actionColorOptions">
   <property name="icon">
    <iconset resource="uiresources.qrc">
     <normaloff>:/icon/icons/Eye Shadows-32.png</normaloff>:/icon/icons/Eye Shadows-32.png</iconset>
   </property>
   <property name="text">
    <string>ColorOptions</string>
   </property>
   <property name="toolTip">
    <string>Color converting options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="uiresources.qrc"/>
 </resources>
 <connections/>
</ui>
